test/handshake_helper.c

```diff
@@ -401,13 +401,131 @@ static void configure_handshake_ssl(SSL *server, SSL *client,
                                  ssl_servername_name(extra->client.servername));
 }
 
-
+/* The status for each connection phase. */
 typedef enum {
     PEER_SUCCESS,
     PEER_RETRY,
     PEER_ERROR
 } peer_status_t;
 
+/* An SSL object and associated read-write buffers. */
+typedef struct {
```

Comment:
okay, here i'm on more solid ground.  :)  456 of 571 have a name for the struct.